lib/gen_stage.ex

```diff
@@ -2427,7 +2427,12 @@ defmodule GenStage do
         {:stop, reason}
     end
 
-    :lists.foldl(fold_fun, {:ok, stage, continue}, producers)
+    :lists.foldl(fold_fun, {:ok, stage, continue_or_hibernate}, producers)
+    |> case do
+      {:ok, stage, :no_continue} -> {:ok, stage}
+      {:ok, stage, continue_or_hibernate} -> {:ok, stage, continue_or_hibernate}
```

Comment:
We don't need this clause either.

Reply:
Right, the `otherwise` is enough. Will chang.e